src/StructuralIdentifiability.jl

```diff
@@ -118,22 +119,22 @@ function assess_identifiability(ode::ODE{P}, funcs_to_check::Array{<:RingElem,1}
     @info "Global identifiability assessed in $runtime seconds"
     _runtime_logger[:glob_time] = runtime
 
-    result = Array{Symbol,1}()
+    result = Array{Pair{Any, Symbol},1}()
```

Comment:
Why do we have an array here, not a dict right away?

Reply:
I was trying to minimize the code changes.

Reply:
Yes, but it looks a bit strange now, maybe we use dict right away, I do not expect this to break anything (and we have tests!)